acceptance/bundle/deploy/dashboard/simple_outside_bundle_root/test.toml

```diff
@@ -0,0 +1,44 @@
+Local = true
+Cloud = true
+RequiresWarehouse = true
+
+Ignore = [
+    "databricks.yml",
+    "sample-dashboard.lvdash.json",
+]
+
+[[Repls]]
+Old = "[0-9a-f]{32}"
+New = "[DASHBOARD_ID]"
+
+[[Repls]]
+# Windows:
+Old = 'The system cannot find the file specified.'
+New = 'no such file or directory'
+
+[[Server]]
+Pattern = "POST /api/2.0/lakeview/dashboards"
+Response.Body = '''
+{
+    "dashboard_id":"1234567890abcdef1234567890abcdef"
+}
+'''
+
+[[Server]]
+Pattern = "POST /api/2.0/lakeview/dashboards/{dashboard_id}/published"
```

Comment:
Question - is this (and other) override needed given that @shreyas-goenka added dashboards to testserver in https://github.com/databricks/cli/pull/3016/files#diff-68abf7b378e7f4092ba8e68a3718d34cb63e3b823eae12382d04b5a8ae33ed2d ?